test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1174,15 +1186,15 @@ UCS_TEST_SKIP_COND_P(test_ucp_sockaddr_wireup_fail,
                      !cm_use_all_devices())
 {
     connect_and_fail_wireup(sender(), FAIL_WIREUP_MSG_SEND,
-                            UCP_EP_FLAG_CONNECT_REQ_QUEUED);
+                            UCP_EP_FLAG_CLIENT_CONNECT_CB);
```

Comment:
why?

Reply:
added comments

Reply:
does it mean the test was not really doing what it was supposed to?

Reply:
> does it mean the test was not really doing what it was supposed to?

yes, the test was broken